frontend/main/mainController.js

```diff
@@ -154,6 +153,21 @@
             return AuthService.emailVerified();
         };
 
+        mainCtrl.updateUser = function updateUser() {
+            UserService.load().then(function success(response) {
+                mainCtrl.user.institutions = response.institutions;
+                mainCtrl.user.follows = response.follows;	
+                mainCtrl.user.institution_profiles = response.institution_profiles;	
+                mainCtrl.user.permissions = response.permissions;
+                AuthService.save();
+            });
+        }
+
+        function notificationListener() {
```

Comment:
rename this function to a better name like startNotificationListerner, or notificationListenerStarter, something like that.